ios/FluentUI/Vnext/Avatar/Avatar.swift

```diff
@@ -148,6 +149,7 @@ public struct AvatarView: View {
         let presenceIconFrameDiffRelativeToOuterRing: CGFloat = ringOuterGapSize - (presenceIconFrameSideRelativeToInnerRing + outerGapAndRingThicknesCombined)
         let presenceCutoutOriginCoordinates: CGFloat = ringOuterGapSize - presenceIconFrameDiffRelativeToOuterRing - presenceIconOutlineSize
         let presenceIconFrameSideRelativeToOuterRing: CGFloat = presenceIconFrameSideRelativeToInnerRing + outerGapAndRingThicknesCombined
+        let overallFrameSide = max(ringOuterGapSize, presenceIconFrameSideRelativeToOuterRing)
```

Comment:
overallFrameSize?

Reply:
The naming of "side" is due to the size being defined by a square that has 4 equal sides or edges (height is always equal to width).
I used "size" for the types defined by the tokens or thickness of the ring or gaps.